CustomCommandExamples/Executor Input Example_1/Executor Input Example_1.cs

```diff
@@ -0,0 +1,35 @@
+using Skyline.DataMiner.Automation;
+
+/// <summary>
+/// DataMiner Script Class.
+/// </summary>
+public class Script
+{
+	/// <summary>
+	/// The Script entry point.
+	/// </summary>
+	/// <param name="engine">Link with SLAutomation process.</param>
+	public void Run(Engine engine)
+	{
+		// This input will not be asked from the user but will automatically be filled in by the DataMiner bot with the
+		// dataminer.services email address of the user executing this automation script using their linked DataMiner account
```

Comment:
When "Automation" refers to the DataMiner Automation module, we usually capitalize it.
```suggestion
		// dataminer.services email address of the user executing this Automation script using their linked DataMiner account
```